source/dem/pw_linear_force.cc

```diff
@@ -95,6 +95,16 @@ PWLinearForce<dim>::PWLinearForce(
       calculate_rolling_resistance_torque =
         &PWLinearForce<dim>::viscous_resistance;
     }
+  if (dem_parameters.boundary_motion.motion_method ==
+      Parameters::Lagrangian::BoundaryMotion<dim>::MotionMethod::free)
+    {
+      this->calculation_force_torque = true;
+    }
+  else
+    {
+      this->calculation_force_torque =
+        dem_parameters.forces_torques.calculate_force_torque;
+    }
```

Comment:
Don't modify boundary motion types for now.

Reply:
ok